src/components/Error/NotFoundPage.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Row, Col, Button } from 'reactstrap';
+import { isExpectedPath } from 'kpmp-common-components';
+ 
+export function getBackToHomePath(pathname) {
+    if(isExpectedPath(pathname, 'explorer')) {
+        return '/explorer'
+    } else if(isExpectedPath(pathname, 'repository')) {
+    	return '/repository'
+    } else {
+        return '/'
+    }
+}
+
+export function getPageTitle(pathname) {
```

Comment:
The same idea would apply here, though we only want to return the string 'KPMP Data Lake Uploader'..

Same point too about needing the function vs getting rid of it.